BPOR/NIHR.NotificationService/Models/SendBatchEmailRequest.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Concurrent;
+
+namespace NIHR.NotificationService.Models
+{
+    public class SendBatchEmailRequest
+    {
+        public IEnumerable<string> EmailAddresses { get; set; }
```

Comment:
I would pair the email with the personalisation data so they don't get mixed up during processing.

Reply:
so currently this is the dictionary key, should we leave it there and just use it?  Doesn't seem very explicit though and if we want to use the notification service and not pass any personalisation data?